### ISSUE TITLE
Prints class names on start

### ISSUE THREAD

**Issue by essarn** [fluttercommunity/get_it#55]
I register a couple of lazy singletons before the app is started. 

```
void setupLocators() {
  locator.registerLazySingleton(() => DatabaseService());

  locator.registerLazySingleton(() => Foo());
  locator.registerLazySingleton(() => Bar());
}
```

```
void main() {
  setupLocators();
  runApp(MultiProvider(
    providers: <SingleChildWidget>[
      ChangeNotifierProvider.value(value: locator<Foo>()),
      ChangeNotifierProvider.value(value: locator<Bar>()),
    ],
    child: App(),
  ));
}
```

But the package seems to print the name of each registered lazy singleton to the console at the same time.

```
I/flutter (21761): DatabaseService
I/flutter (21761): Foo
I/flutter (21761): Bar
```

**Comment by JonasWanke** [comment]
There's already a PR for this issue: #50

**Comment by essarn** [comment]
Sorry for not checking the previous issues or current pull requests.

**Comment by escamoteur** [comment]
Fixed now in V4.0.1
